x/supplier/types/message_create_claim.go

```diff
@@ -47,27 +47,27 @@ func (msg *MsgCreateClaim) GetSignBytes() []byte {
 
 func (msg *MsgCreateClaim) ValidateBasic() error {
 	// Validate the supplier address
-	_, err := sdk.AccAddressFromBech32(msg.SupplierAddress)
+	_, err := sdk.AccAddressFromBech32(msg.GetSupplierAddress())
 	if err != nil {
-		return sdkerrors.Wrapf(ErrSupplierInvalidAddress, "invalid supplierAddress address (%s)", err)
+		return sdkerrors.Wrapf(ErrSupplierInvalidAddress, "%s", msg.GetSupplierAddress())
```

Comment:
```suggestion
		return ErrSupplierInvalidAddress.Wrapf("%s", msg.GetSupplierAddress())
```
Same for all the others

Reply:
Thanks for calling this out! :raised_hands: 

I thought I got all of these but I either missed some or there were some reversions between the cherry-picking, etc.

Reply:
After looking into this for second:

![image](https://github.com/pokt-network/poktroll/assets/600733/4297f9bf-4a19-43eb-8d5f-d8364bde99ec)

It seems like we have lots of usages of `sdkerrors.Wrapf()` across the modules (none of which are in tests). This leads me to believe that this is baked in to the ignite generator template (in whatever ignite version was used). 

I would prefer to batch this out in a separate PR **after v0.50 migration**, in case we do decide to transplant regenerate module code.

Reply:
Captured in #347.